.env.template

```diff
@@ -22,4 +22,17 @@ TRELLO_BOARD_NAME_CREATE = board_to_be_created
 LOG_LEVEL=DEBUG
 
 # Loggly token needs to be fetched from Loggly to authenticate the account. 
-LOGGLY_TOKEN=loggly-token-value
\ No newline at end of file
+LOGGLY_TOKEN=loggly-token-value
+
+# Mongo DB Authentication variables
+MONGO_USER_NAME=user-name
+MONGO_PASSWORD=password
+MONGO_URL=mongo-cluster-name
+
+# Mongo DB variables to connect to production database and collection. 
+TODO_DATABASE=Todo_Database
+TODO_COLLECTION=Todo_Collection
+
+# Mongo DB test variables to run integration and E2E tests
+TEST_DATABASE=Test-database
+TEST_COLLECTION=Test-collection
```

Comment:
Can you add the new variables (`OAUTH_CLIENT_ID` and `OAUTH_CLIENT_SECRET`) to here?

Reply:
Done!